packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -181,20 +184,25 @@ export const DatepickerCalendar: ComponentWithAs<'div', DatepickerCalendarProps>
     setGridNavigatedDate(updatedGridNavigatedDate);
   };
 
+  const prevMonthOutOfBounds = minDate ? compareDatePart(minDate, getMonthStart(gridNavigatedDate)) >= 0 : false;
+  const nextMonthOutOfBounds = maxDate ? compareDatePart(getMonthEnd(gridNavigatedDate), maxDate) >= 0 : false;
+
   const handleKeyDown = (e, day) => {
     const keyCode = getCode(e);
     const initialDate = day.originalDate;
     let targetDate: Date | null = null;
+    const visuallyPleasingNavigation = false;
```

Comment:
Love the name, but could it be a bit more self-explanatory? :) 
What is visually pleasing? :)

Reply:
Renamed to `visuallyPleasingWeekJumpNavigation`. This is the navigation behavior that we'll be discussing with the designer and PM.